app/models/lockstep/magic_link_summary.rb

```diff
@@ -0,0 +1,16 @@
+class Lockstep::MagicLinkSummary < Lockstep::ApiRecord
+  self.model_name_uri = "v1/useraccounts/magic-links/summary"
+  self.id_ref = "group_key"
+  self.query_path = ""
+  load_schema(Schema::MagicLinkSummary)
+
+  def self.with_date(from_date: nil, to_date: nil)
```

Comment:
If we send nil date, is it being handled by platform correctly?

Reply:
Yes. It sets from to one year before and to date to current date